packages/devtools_app/lib/src/shared/preferences/_inspector_preferences.dart

```diff
@@ -373,10 +412,21 @@ class InspectorPreferencesController extends DisposableController
   }
 
   /// Change the value for the hover eval mode setting.
+  ///
+  /// Note: This is only used for testing purposes.
```

Comment:
nit: this note is redundant with the visibleForTesting tag. Can we delete this note or delete the dart doc altogether since the dart doc is kind of redundant itself?

Reply:
Done